admin_auto_filters/filters.py

```diff
@@ -88,7 +97,8 @@ def get_form_field(self):
     def _add_media(self, model_admin, widget):
 
         if not hasattr(model_admin, 'Media'):
-            raise ImproperlyConfigured('Add empty Media class to %s. Sorry about this bug.' % model_admin)
+            model_admin.__class__.Media = type('Media', (object,), dict())
+            model_admin.__class__.media = media_property(model_admin.__class__)
```

Comment:
This is really nice!

Reply:
awesome man !

Reply:
Thanks! I think this works, at least. 😉 I did add a test, but it's entirely possible that this is deficient in some way - this is the only time I've ever customized the Media class. Also, I did see #33 - not sure if there's anything there which doesn't get addressed here.